cli/commands/const.go

```diff
@@ -0,0 +1,14 @@
+package commands
+
+import "time"
+
+// DEFAULT VALUES
+const (
+	defaultStartTimeout = 1 * time.Minute
```

Comment:
It's not "common" values. I think it could be easily separated to start/const.go and log/const.go.

But you can argue with me.

Reply:
I sure that this default values are about parsing commands and should be declared in the `commands` package.